docs/reference/search/profile.asciidoc

```diff
@@ -276,14 +292,22 @@ the equivalent name in Elasticsearch.  The `"description"` field displays the Lu
 is made available to help differentiating between parts of your query (e.g. both `"message:search"` and `"message:test"`
 are TermQuery's and would appear identical otherwise.
 
-The `"time"` field shows that this query took ~15ms for the entire BooleanQuery to execute.  The recorded time is inclusive
+The `"time"` field shows that this query took ~1.8ms for the entire BooleanQuery to execute. The recorded time is inclusive
+of all children.
+
+The `"time_in nanos"` field shows the timing information in an exact, machine readable format in nanoseconds.
 of all children.
 
 The `"breakdown"` field will give detailed stats about how the time was spent, we'll look at
 that in a moment.  Finally, the `"children"` array lists any sub-queries that may be present.  Because we searched for two
 values ("search test"), our BooleanQuery holds two children TermQueries.  They have identical information (type, time,
 breakdown, etc).  Children are allowed to have their own children.
 
+NOTE: The `"time"` field is only intended for human consumption. If you need exact timing values use `"time_in nanos"`. The `"time"`
```

Comment:
I'm only following the way the field names are written in the previous paragraph here. It's not completely consitent in this part of the docs I think. Not sure if I should make this change bigger by changing it one way or the other in the whole document?